docs/go/quickstart/quickstart.md

```diff
@@ -11,6 +11,7 @@ tags:
   - hello world
 ---
 
-- [URL Shortener with Echo-SQL](/docs/go/quickstart/echo-sql) - A sample URL Shortener app that responds with a shortened URL of the URL sent as a request. The application tests Keploy integration capabilities using [Echo](https://echo.labstack.com/) and [PostgreSQL](https://www.postgresql.org/). 
+- [URL Shortener with Echo-SQL](/docs/go/quickstart/echo-sql) - A sample URL Shortener app that responds with a shortened URL of the URL sent as a request. The application tests Keploy integration capabilities using [Echo](https://echo.labstack.com/) and [PostgreSQL](https://www.postgresql.org/).
 - [URL Shortener with Gin-Mongo](/docs/go/quickstart/gin-mongo) - A sample application that does CRUD (get, create, update, and delete) operations on a user profile in [mongoDB](https://www.mongodb.com/).
 - [User Profile with Gin-Mongo](/docs/go/quickstart/gin-mongo-2) - A sample URL Shortener app that responds with a shortened URL of the URL sent as a request. The application tests Keploy integration capabilities using [Gin](https://gin-gonic.com) and [mongoDB](https://www.mongodb.com/).
+- [Contact Keeper with Gorilla/Mux-Redis](/docs/go/quickstart/gorillamux-redis): A sample contact keeper app to test Keploy integration capabilities using Gorilla/Mux and Redis.
```

Comment:
@developer-diganta can you add a one line detail of does the app does, like I can see it stores the contact of the user.